Source/Core/DolphinQt/Debugger/CodeViewWidget.cpp

```diff
@@ -208,6 +209,55 @@ void CodeViewWidget::ReplaceAddress(u32 address, ReplaceWith replace)
   Update();
 }
 
+bool CodeViewWidget::IsInstructionLoadStore(std::string instruction)
+{
+  // Could add check for context address being near PC, because we need gprs to be correct for the
+  // load/store.
+  if ((instruction.compare(0, 2, "st") != 0 && instruction.compare(0, 1, "l") != 0 &&
+       instruction.compare(0, 5, "psq_l") != 0 && instruction.compare(0, 5, "psq_s") != 0) ||
+      instruction.compare(0, 2, "li") == 0)
+    return false;
+  else
+    return true;
+}
+
+void CodeViewWidget::OnCopyTargetAddress()
+{
+  const std::string codeline = PowerPC::debug_interface.Disassemble(GetContextAddress());
+
+  if (!IsInstructionLoadStore(codeline))
+    return;
+
+  const u32 addr = PowerPC::debug_interface.GetMemoryAddressFromInstruction(codeline);
```

Comment:
```suggestion
  const u32 address = PowerPC::debug_interface.GetMemoryAddressFromInstruction(codeline);
```

You don't need to mince words, we aren't running on an interpreter on a system with so little memory that shortening names is needed to save memory ;)

Reply:
I don't mind making it longer, but all the other functions use addr instead of address, so I probably should leave it for consistency?